<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGA2ZYZClass</class>
 <widget class="QWidget" name="GGA2ZYZClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GGA2ZYZ</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_EditFilePath">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_ButtonSelect">
       <property name="text">
        <string>打开文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="listWidget"/>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QLineEdit" name="lat"/>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="h"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="toolTip">
          <string>筛选最大差分龄期，延迟大于输入秒数不输出。</string>
         </property>
         <property name="text">
          <string>延迟限制：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLineEdit" name="breakPeriod">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="toolTip">
          <string>选择输出所有状态还是只有固定解。</string>
         </property>
         <property name="text">
          <string>解状态：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>开始时间：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="statement">
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>ALL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GGA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PPPNAVA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BESTPOSA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>INSPVAXA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RTKPOSA</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>循环周期：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="toolTip">
          <string>筛选数据的时间间隔，每隔指定时间输出一个。</string>
         </property>
         <property name="text">
          <string>间隔</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>中断时间：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>中央子午线：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="meridian">
         <property name="text">
          <string>114</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lon"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="interval">
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>auto</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>60s</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="solutionStatus">
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>fixed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>all</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>h</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>语句</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="timeSlice">
         <property name="text">
          <string>60</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="startTime">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="delayLimit">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="toolTip">
          <string>筛选每分钟前多少秒的数据。</string>
         </property>
         <property name="text">
          <string>时间段：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLineEdit" name="cycleTime">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="logText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_btn">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="time_lb">
       <property name="text">
        <string>00:00:00:000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="kml">
       <property name="text">
        <string>kml</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="planarProjection">
       <property name="text">
        <string>平面投影</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="comparison">
       <property name="text">
        <string>对比图</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
